### PR TITLE
Lambdas detail fiorification

### PR DESCRIPTION
**Pull request by maxmarkus**
applied fiori to detail page, its direct child components, without touching its modals

### CODE REVIEW

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.ts [hunk 1]
```diff
@@ -10,22 +7,33 @@ import {
...
+
 import 'brace';
 import 'brace/ext/language_tools';
 import 'brace/snippets/javascript';
 import 'brace/snippets/json';
 import 'brace/snippets/text';
 import 'brace/mode/javascript';
 import 'brace/mode/json';
-import 'brace/theme/eclipse';
+// import 'brace/theme/chrome';
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.ts]: I have reviewed EVERY theme and these are all themes that could fit to our design. Therefore I would keep it since we might want to exchange it in the future and do not have to search again through all themes. I will leave a comment instead.

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html [hunk 2]
```diff
@@ -1,41 +1,58 @@
...
+          [disabled]="!lambdaName || isLambdaNameInvalid" (click)="showBindingCreatorModal()">
+          Add
+        </button>
+      </fd-panel-actions>
+    </fd-panel-header>
+  </fd-panel>
 
+  <fd-panel *ngIf="instanceBindingInfoList.length > 0">
+    <fd-panel-body>
+      <table class="fd-table">
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: Apart from environment variables list I pointed out below, the table doesn't look good.
The same problem happens at the bottom.
And the same happens in create mode when environment variables list is empty.
image

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html [hunk 3]
```diff
@@ -1,199 +1,226 @@
...
+                </li>
+              </fd-menu-list>
+            </fd-menu>
+          </fd-popover-body>
+        </fd-popover>
+      </fd-panel-actions>
+    </fd-panel-header>
+
+    <fd-panel-filters>
+      <!-- trigger badges -->
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: I think vertical-align:middle for the badge would look much better here.
image

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html [hunk 4]
```diff
@@ -1,199 +1,226 @@
...
-                        <a [class.disabled]="lambda.metadata.name.length <= 0 || isFunctionNameInvalid || isHTTPTriggerAdded"
-                           (click)="showHTTPTrigger()" class="tn-dropdown__item">Expose via HTTPS</a>
-                      </li>
-                    </ul>
+  
+  <div *ngIf="(loaded | async) === true">
+    <fd-panel>
+      <fd-panel-header>
+        <fd-panel-head>
+          <fd-panel-title>General informations</fd-panel-title>
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: *information

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html [hunk 5]
```diff
@@ -1,199 +1,226 @@
...
+      </fd-panel-body>
+    </fd-panel>
+  </div>
+</section>
+
+<section class="fd-section">
+  <fd-panel>
+    <fd-panel-header>
+      <fd-panel-head>
+        <fd-panel-title>General informations</fd-panel-title>
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: *information. Should this header be the same as in the section above?

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html (hunk 4 above)
Comment (dariadomagala) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: Why do we have it in two separate panels that have the same title? Why not have it together as it used to be?

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html [hunk 6]
```diff
@@ -1,41 +1,58 @@
...
+        <thead>
+          <tr>
+            <td class="y-fd-table__th--col-3">Instance</td>
```
Comment (dariadomagala) [inline lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: Could we make it a bit bigger? ;) 
Like col-4 or col-5?

File: lambda/src/app/lambdas/list/lambdas-entry-renderer/lambdas-entry-renderer.component.html [hunk 7]
```diff
@@ -1,10 +1,39 @@
+<tr>
```
Comment (dariadomagala) [inline lambda/src/app/lambdas/list/lambdas-entry-renderer/lambdas-entry-renderer.component.html]: Why did you change lists views? This task was about details view only. This file has been changed here https://github.com/jesusreal/console/pull/54/files#diff-84e15d64ab7c33356c07048fa0683172

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html [hunk 8]
```diff
@@ -1,199 +1,226 @@
...
-          </div>
-        </div>
-      </div>
-      <div [hidden]="(hasDependencies | async) === false ">
-        <ace-editor #dependencyEditor [(text)]="dependency" [mode]="aceDependencyMode" [theme]="theme" (textChanged)="onDependencyChange($event)"></ace-editor>
-      </div>
-    </div>
-  </div>
+        <button fd-button [options]="'light'" [glyph]="'less'"
+          *ngIf="(hasDependencies | async)" class="tn-button sf-button--danger tn-button--small tn-button--text"
```
Comment (dariadomagala) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: Please change those classes.

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html [hunk 9]
```diff
@@ -1,41 +1,58 @@
...
+            <td class="y-fd-table__th--col-1"></td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr *ngFor="let instanceBinding of instanceBindingInfoList; let i = index; last as isLast">
+            <td class="y-fd-table__th--col-3">
+              {{ instanceBinding.instanceName }}
+            </td>
+            <td class="y-fd-table__th--col-8">
+              <table>
```
[the remark this answers is not in the document]
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: I only touched the classes, not the markup here. But I agree that it could be aligned. Should be done in phase 2.

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html (hunk 5 above)
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: Oh yes, wrong header title.

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html (hunk 2 above)
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: Cannot reproduce this issue, looks fine for me in Chrome, FF, Safari.

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (parostatkiem) [reply lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: Ok, please add a comment on Phase 2 task.

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.ts [hunk 10]
```diff
@@ -59,8 +66,7 @@ const FUNCTION = 'function';
   styleUrls: ['./lambda-details.component.scss'],
 })
 @HostListener('sf-content')
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-details.component.ts]: Should it be here?

File: lambda/src/app/lambdas/lambda-details/lambda-env/lambda-env.component.html [hunk 11]
```diff
@@ -1,43 +1,55 @@
...
+            </fd-form-message>
+            <fd-form-message [type]="'warning'" *ngIf="isEnvVariableDuplicate">
+              Duplicate entry.
+            </fd-form-message>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let env of envs; let i = index">
+          <td class="y-fd-table__th--col-4">{{ env.name }}</td>
```
Comment (parostatkiem) [inline lambda/src/app/lambdas/lambda-details/lambda-env/lambda-env.component.html]: There's no need to use column-width classes on `<td>`. You already defined it above in `<th>` :)

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html (hunk 2 above)
Comment (parostatkiem) [reply lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: So we're going to change this part anyway and I shouldn't care about this bug?

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.html (hunk 4 above)
Comment (parostatkiem) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.html]: Still unresolved...

File: lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html (hunk 2 above)
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-instance-bindings/lambda-instance-bindings.component.html]: Reproduced and fixed.

File: lambda/src/app/lambdas/lambda-details/lambda-details.component.ts (hunk 10 above)
Comment (maxmarkus) [reply lambda/src/app/lambdas/lambda-details/lambda-details.component.ts]: That is something I also have seen and wondered. I checked and it seems it is not needed.

**Comment by maxmarkus** [comment]
I have now fixed the word-break issue for the two tables and removed this span _Edit Trigger_ title.
